sdk/src/Core/Amazon.Runtime/Internal/Auth/AWS4Signer.cs

```diff
@@ -694,16 +687,6 @@ public static string DetermineSigningRegion(IClientConfig clientConfig,
             var endpoint = clientConfig.RegionEndpoint;
             if (endpoint != null)
             {
-#pragma warning disable CS0612,CS0618
-                var serviceEndpoint = endpoint.GetEndpointForService(serviceName, clientConfig.ToGetEndpointForServiceOptions());
-                if (!string.IsNullOrEmpty(serviceEndpoint.AuthRegion))
```

Comment:
It was safe to remove the `serviceEndpoint.AuthRegion != null` check because the endpoint returned from `GetEndpointForService` sets the auth region to null.